src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1453,84 +1567,68 @@ namespace Google.Example.Library.V1.Samples
     {
         /// <summary>
-        /// Testing &lt;@calling forms&gt;
+        /// Testing calling forms
         /// </summary>
-        /// <param name="shelfName">The name of the shelf where books are published to.</param>
-        /// <param name="edition">The edition of the series.</param>
-        public static async Task SamplePublishSeriesAsync(string shelfName, uint edition)
+        public static void SampleGetBigBook(string shelf)
         {
-            LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-            // string shelfName = "Math"
-            // uint edition = 123
-            Shelf shelf = new Shelf
-            {
-                Name = "Math",
-            };
-            IEnumerable<Book> books = new List<Book>();
-            uint edition = 123;
-            SeriesUuid seriesUuid = new SeriesUuid
-            {
-                SeriesString = "xyz3141592654",
-            };
-            PublishSeriesResponse response = await libraryServiceClient.PublishSeriesAsync(shelf, books, edition, seriesUuid);
-            // % % % output handling % % % %
-            // fourScoreAndSevenYears ago
-            //
-            // our fathers brought forth upon this continent
-            //
-            // a new nation
-            // conceived in liberty
-            //
-            // and dedicated to the proposition that allMenAreCreatedEqual.
-            //
-            // Do something with bookNames one by one
-            IEnumerable<string> bookNames = response.BookNames;
-            foreach (var title in response.BookNames) {
-                // Now we deal with thisSingleBook!
-                Console.WriteLine($"\t%`` The book's title: \"{title}\", \\\nand the book costs $50.00 ``%");
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // string shelf = "Novel\\\"`\b\t\n\r"
+            BookName name = new BookName(shelf, "War and Peace");
+            // Poll until the returned long-running operation is complete
+            Book response = libraryServiceClient.GetBigBook(name).PollUntilCompleted().Result;
+            // Testing iterating over map fields when both key and value are specified.
+            foreach (KeyValuePair<string, ListValue> entry in response.MapListValueValue.entrySet()) {
+                string myKey = entry.Key;
+                ListValue myValue = entry.Value;
+                Console.WriteLine($"key: {myKey}, value: {myValue}");
             }
-            Console.WriteLine($"The first book is: {response.BookNames[0]}");
-            Console.WriteLine($"The author of the first book is: {response.Books[0].Author}");
-            Console.WriteLine("That's all!");
-            Console.WriteLine($"series_uuid: {response.SeriesUuid.SeriesBytes}");
+            // Testing iterating over map fields when only key is specified.
+            foreach (var anotherKey in response.MapListValueValue.Keys) {
```

Comment:
If it's possible to replace the `var` with the actual type that'd be better.

Reply:
Done